src/openforms/forms/api/serializers/form.py

```diff
@@ -114,6 +113,16 @@ class FormSerializer(serializers.ModelSerializer):
     is_deleted = serializers.BooleanField(source="_is_deleted", required=False)
     required_fields_with_asterisk = serializers.SerializerMethodField(read_only=True)
 
+    translations = ModelTranslationsSerializer(
```

Comment:
@sergei-maertens currently `translations` for the `Form` endpoint are spread across two places:
1. at the top-level: `previousText`, `confirmText`, `changeText`, `beginText`, `explanationTemplate`, `submissionConfirmationTemplate` and `name`
2. at the `confirmationEmailTemplate` level (because this is a separate model): `subject`, content`

Is this okay? Do we want everything to be at the top-level, or do we want everything as close to the location of the original values? (E.g. do we want the literal translations under `literals.translations`?)

Reply:
I'd say top level to keep translations isolated from the non-translatable data (without thinking too much on how to implement this :grimacing: )